src/main/resources/liquibase/workbench_changelog/14_3_0.xml

```diff
@@ -49,5 +49,25 @@
 			insert into role_type_permission(role_type_id, permission_id, selectable) values (2, @last_permission_id , 1);
         </sql>
     </changeSet>
+    <changeSet author="cuenyad" id="v14.3.0-3">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="0">
+                SELECT count(*) FROM permission
```

Comment:
A new permission is not needed. According to the discussions during the grooming and estimations, the View transations permission will automatically enable this feature. @cuenyad